build.sh

```diff
@@ -39,7 +39,7 @@ fi
 # build core part
 CORE_DIR="${CURRENT_DIR}/core"
 pushd ${CORE_DIR}
-mvn clean package -q -DskipTests
+mvn clean package -q
 popd # core dir
```

Comment:
Could we add an option or environ parameter to disable the tests? Because we should add the UT tests for the core part, and that would be not necessary for people who want to build the project with themselves.

Reply:
OK